draft-irtf-cfrg-vdaf.md

```diff
@@ -1673,6 +1673,48 @@ def decode_vec(Field, encoded: Bytes) -> Vec[Field]:
 ~~~
 {: #field-derived-methods title="Derived class methods for finite fields."}
 
+Finally, it is useful to define class methods on `Field` to:
+
+* encode an unsigned integer into its bit vector representation, expressed
+  as a vector of field elements.
+* decode a field element from a bit vector representation.
+
+~~~
+def encode_into_bit_vector(Field,
+                           val: Unsigned,
```

Comment:
It is a bit odd. The reason is when we encode integer into bit vector, the measurement is always in `Unsigned` form. When we `truncate`, we need to decode the vector into a `Field`, because the `Field` is actually secret-shared. I could instead make the function name more explicit, e.g. `encode_integer_into_bit_vector` and `decode_field_from_bit_vector`?

Reply:
Hmm, fair. I think I'd still prefer to take in a `Field`, even if that means we need to  `Field.encode_into_bit_vector(elem=Field(23))` then call `elem.as_unsigned()` in the encoding method internally. But I don't feel strongly about it. Feel free to leave as-is, and we'll see if maybe @divergentdave has a strong opinion here.